.travis.yml

```diff
@@ -11,8 +11,12 @@ env:
   global:
     - GO111MODULE=on
     - PATH=${TRAVIS_BUILD_DIR}/bin:${PATH}:/usr/local/go/bin
+    - GOROOT=
```

Comment:
GOROOT is uninitialized? This should point to /usr/local/go/bin ?

Reply:
This variable is not required by go. I'm doing this here because travis sets it  to path for go1.11.1 direcotry

Reply:
What is the use of this just definition without proper assignment? Let me know if there is any specific reason for this statement. Either set to default  GOROOT="/usr/local/go" or remove it to use the default path, but don't define without proper assignment.

Reply:
The use of this statement is to make the variable empty (it is set by travis) and force go to use default configuration. This variable is set by travis to point to /home/travis/.gimme/versions/go1.11.1.linux.amd64 path but since we want 1.15.5 version I'm deleting this directory.